presentation/src/main/java/com/example/healthysalad/MainActivity.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.healthysalad
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.example.data.Data
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        Data().loadAllItems()
```

Comment:
`Data` instance can be used in other method like onResume(), onStart(), right?
If it is, you should store the Data instance in the member variable.